src/Glpi/Controller/Form/Utils/CanCheckAccessPolicies.php

```diff
@@ -62,6 +62,9 @@ protected function checkFormAccessPolicies(Form $form, Request $request): void
 
```

Comment:
```suggestion
        // If the user is not logged in and the form require a valid session,
        // redirect him to the login page instead.
        // Note that the session validity will still be checked by the `canAnswerForm`
        // method right below, this code is only here to prevent a permission error
        // and to redirect the user to the login page.
        if (
            !Session::isAuthenticated()
            && !$form_access_manager->allowUnauthenticatedAccess($form)
        ) {
            // Will trigger a redirection to the loggin page
            throw new SessionExpiredException();
        }
```

Reply:
Thanks for taking a look; I will get the suggested changes added and update the request